toolkit/authentication/src/main/java/com/arcgismaps/toolkit/authentication/ServerTrustAuthenticator.kt

```diff
@@ -119,11 +122,29 @@ private fun ServerTrustAuthenticatorImpl(
     Column(
         modifier = modifier
             .padding(24.dp),
-        horizontalAlignment = Alignment.CenterHorizontally,
         verticalArrangement = Arrangement.Center
     ) {
         Text(
-            text = stringResource(id = R.string.server_trust_message, hostname),
+            text = stringResource(id = R.string.server_trust_title),
+            style = MaterialTheme.typography.headlineSmall,
+            textAlign = TextAlign.Start
+        )
+        Spacer(modifier = Modifier.height(16.dp))
+        Text(
+            text = buildAnnotatedString {
+                val string = stringResource(
+                    id = R.string.server_trust_message,
+                    hostname
+                )
+                val startIdx = string.indexOf(hostname)
+                val endIdx = startIdx + hostname.length
+                append(string)
+                addStyle(
+                    SpanStyle(
+                        fontWeight = FontWeight.Bold
+                    ), startIdx, endIdx
```

Comment:
Slight formatting suggestion: 
```suggestion
                    style = SpanStyle(fontWeight = FontWeight.Bold),
                    start = startIdx,
                    end = endIdx
```